apps/src/music/views/SoundsPanel.tsx

```diff
@@ -162,10 +184,10 @@ const SoundsPanelRow: React.FunctionComponent<SoundsPanelRowProps> = ({
         styles.soundRow,
         isSelected && styles.soundRowSelected
       )}
-      onClick={() => onSelect(folder.id + '/' + sound.src)}
+      onClick={onSoundClick}
       onKeyDown={event => {
         if (event.key === 'Enter') {
-          onSelect(folder.id + '/' + sound.src);
+          onSoundSelect();
```

Comment:
Just confirming, you don't need to pass `folder.id + '/' + sound.src` to these functions because `soundPath` already stores this value right?

Reply:
Yes.